c2corg_ui/static/js/feed.js

```diff
@@ -176,6 +184,18 @@ app.FeedController.prototype.toggleFilters = function() {
   window.scrollTo(0, 0);
 };
 
+
+/**
+ * @param {string} filename
+ * @param {string} suffix
+ * @return {string | undefined}
```

Comment:
Why "undefined"?
The function at least returns a string starting with ``this.imageUrl_``.